src/ui_manager.cpp

```diff
@@ -421,14 +440,22 @@ void ui_adaptor::redraw_invalidated()
                         ui.invalidated = false;
                     }
                 }
+                if( ui.is_imgui && draw_imgui && ( !ui.is_imgui || imgui_is_on_top ) ) {
```

Comment:
This is equivalent to
```suggestion
                if( ui.is_imgui && draw_imgui && imgui_is_on_top ) {
```

Reply:
done